app/utils/premium.ts

```
@@ -26,6 +26,26 @@ export function getHasPremium(roles: string[] | undefined | null): boolean {
+
+/**
+ * Checks if Discord roles need to be refreshed based on the last refresh timestamp
+ * @param refreshedAt - Timestamp when roles were last refreshed (string)
+ * @returns true if roles need refreshing, false otherwise
+ */
+export function needsRolesRefresh(
+  refreshedAt: string | undefined | null
+): boolean {
+  if (!refreshedAt) return true
+
+  const lastRefresh = parseInt(refreshedAt, 10)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)